bindings/netframework/ElectionGuard.NetFramework/ElectionGuardInterop/Ballot.h

```diff
@@ -179,6 +259,156 @@ namespace ElectionGuardInterop
                   return elements;
               }
           }
+
+          bool
+          isValid(String ^ expectedObjectId, uint64_t expectedNumberSelections,
+                  uint64_t expectedNumberElected)
+        {
+            std::string _expectedObjectId;
+            Utilities::MarshalString(expectedObjectId, _expectedObjectId);
+
+            return this->_instance->isValid(_expectedObjectId, expectedNumberSelections,
+                                            expectedNumberElected);
+        }
+
+        bool isValid(String ^ expectedObjectId, uint64_t expectedNumberSelections,
+                     uint64_t expectedNumberElected, uint64_t votesAllowed)
+        {
+            std::string _expectedObjectId;
+            Utilities::MarshalString(expectedObjectId, _expectedObjectId);
+
+            return this->_instance->isValid(_expectedObjectId, expectedNumberSelections,
+                                            expectedNumberElected, votesAllowed);
+        }
+    };
+
+  public
+    ref class CiphertextBallotContest : ManagedInstance<electionguard::CiphertextBallotContest>
```

Comment:
these classes are primarily for property access to traverse the object graph after a ballot has been encrypted.  I did expose the constructors publicly but it's not likely you will ever need to programmatically create a ciphertext ballot contest or a ciphertext selection.